packages/target/README.md

```diff
@@ -0,0 +1,201 @@
+
+# React Native AEP Target Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aeptarget.svg)](https://www.npmjs.com/package/@adobe/react-native-aeptarget) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aeptarget)](https://www.npmjs.com/package/@adobe/react-native-aeptarget)
+
+`@adobe/react-native-aeptarget` is a wrapper around the iOS and Android [AEP Target SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target) to allow for integration with React Native applications. Functionality to enable Adobe Target is provided entirely through JavaScript documented below.
+
+
+## Installation
+
+You need to install the SDK with [npm](https://www.npmjs.com/) and configure the native Android/iOS project in your react native project. Before installing the Target extension it is recommended to begin by installing the [Core extension](https://www.npmjs.com/package/react-native-aep-mobile-core).
+
+> Note: If you are new to React Native we suggest you follow the [React Native Getting Started](<https://facebook.github.io/react-native/docs/getting-started.html>) page before continuing.
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page 
+
+Install the `@adobe/react-native-aeptarget` package:
+
+```bash
+cd MyReactApp
+npm install @adobe/react-native-aeptarget
+```
+
+## Usage
+
+### [Target](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target)
+
+#### Importing the extension:
+```javascript
+import {Target} from '@adobe/react-native-aeptarget';
+```
+
```

Comment:
Let revisit the readme to make the format same as other extensions.
We have a section for
## API reference


Using format

**Syntax**

**Example**